x-pack/plugins/monitoring/server/plugin.ts

```diff
@@ -141,9 +143,14 @@ export class Plugin {
 
     // Initialize telemetry
     if (plugins.telemetryCollectionManager) {
-      registerMonitoringCollection(plugins.telemetryCollectionManager, this.cluster, {
-        maxBucketSize: config.ui.max_bucket_size,
-      });
+      registerMonitoringCollection(
+        plugins.telemetryCollectionManager,
+        this.cluster,
+        () => this.elasticsearchClient,
```

Comment:
I got the `monitoring` collection to work by adding the piece of code to the `start` method in this plugin:

```typescript
  start({ elasticsearch }: CoreStart) {
    // TODO: For the telemetry plugin to work, we need to provide the new ES client.
    //  It should be initialised with a similar config to `this.cluster` but, since not used in
    //  the Monitoring Telemetry collection yet, setting the local client for progress now and we'll
    //  update it in a follow up PR.
    this.elasticsearchClient = elasticsearch.client;
  }
```

Reply:
I'm going to try that. Thanks!

Reply:
It works!